### ISSUE TITLE
📦 Feature - List supported message styling features

### ISSUE THREAD

**Issue by Stevetec-exe** [Valour-Software/Valour#178]
**Is your feature request related to a problem? Please describe.**
As a first time Valour user i wanted to test, what styling-features are supported, but I did/do not want to spam testcases against the system.

**Describe the solution you'd like**
A list of supported features that is shown to new users and a 'sandbox'-channel that only renders pages to the user sending them, a bi like the editing sandbox of Wikipedia (where you can test editing-features on a dummy article)

**Describe alternatives you've considered**
> If it is *just* markdown with no additional features a note or popup along the lines of **'Valour supports markdown formatting'** would be fine.

Fallback: a simple text file on Github.

**Additional context**
-/-

**Comment by drop-lt** [comment]
so like a sandbox thing?

**Comment by Stevetec-exe** [comment]
Yes, exactly. I don't think that you can expect everyone to know Markdown, so a "guided tour" and/or sandbox to show all the features would go a long way for those users

**Comment by drop-lt** [comment]
ok, gonna see what i can do for it

**Comment by Coca162** [comment]
The need for a sandbox no longer exists due to message previewing however some documentation of markdown would still be benefitial
